Building for tvOS fails

tvOS doesn't have a WebKit framework, so creating a separate target might be needed.
![screen shot 2017-10-04 at 1 17 15 pm](https://user-images.githubusercontent.com/1402918/31192251-5ff1249c-a906-11e7-871a-ca7e21a782f4.png)

Getting a similar issue when running `carthage update`.  In the error logs, I get 
```
~project/Carthage/Checkouts/Down/Source/Views/DownView.swift:9:8: error: no such module 'WebKit'
import WebKit
       ^
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,10 +1,12 @@
 osx_image: xcode9
 language: objective-c
 before_install:
+  - set -o pipefail 
   - gem install xcpretty slather -N
   - xcrun instruments -w "iPhone 6 (11.0)" || echo "(Pre)Launched the simulator."
 script:
-  - set -o pipefail && xcodebuild -project Down.xcodeproj -scheme "Down" -destination "platform=iOS Simulator,name=iPhone 6,OS=11.0" -enableCodeCoverage YES ONLY_ACTIVE_ARCH=YES test | xcpretty -c
-  - set -o pipefail && xcodebuild -project Down.xcodeproj -scheme "Down" -sdk macosx -destination 'platform=OS X,arch=x86_64' -enableCodeCoverage YES test | xcpretty -c
+  - xcodebuild -project Down.xcodeproj -scheme "Down" -destination "platform=iOS Simulator,name=iPhone 6,OS=11.0" -enableCodeCoverage YES ONLY_ACTIVE_ARCH=YES test | xcpretty -c
+  - xcodebuild -project Down.xcodeproj -scheme "Down" -sdk macosx -destination 'platform=OS X,arch=x86_64' -enableCodeCoverage YES test | xcpretty -c
+  - xcodebuild -project Down.xcodeproj -scheme "Down" -sdk appletvsimulator -destination 'platform=tvOS Simulator,name=Apple TV' -enableCodeCoverage YES test | xcpretty -c
 after_success:
   - slather coverage --ignore "../**/*/Xcode*" --ignore "Source/cmark/*" --scheme "Down" Down.xcodeproj

--- a/Down.podspec
+++ b/Down.podspec
@@ -1,7 +1,7 @@
 Pod::Spec.new do |spec|
   spec.name         = "Down"
   spec.summary      = "Blazing fast Markdown rendering in Swift, built upon cmark."
-  spec.version      = "0.4.1"
+  spec.version      = "0.4.2"
   spec.homepage     = "https://github.com/iwasrobbed/Down"
   spec.license      = { :type => "MIT", :file => "LICENSE" }
   spec.authors      = { "Rob Phillips" => "rob@robphillips.me" }

--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ Is your app using it? [Let me know!](mailto:rob@robphillips.me)
 
 ### Installation
 
-Note: Swift 4 support is now on the `master` branch.
+Note: Swift 4 support is now on the `master` branch and any tag >= 0.4.x (Swift 3 is 0.3.x)
 
 Quickly install using [CocoaPods](https://cocoapods.org): 
 

--- a/Source/Views/DownView.swift
+++ b/Source/Views/DownView.swift
@@ -6,6 +6,9 @@
 //  Copyright © 2016 Glazed Donut, LLC. All rights reserved.
 //
 
+#if os(tvOS)
+    // Sorry, not available for tvOS
+#else
 import WebKit
 
 // MARK: - Public API
@@ -119,3 +122,4 @@ extension DownView: WKNavigationDelegate {
     }
     
 }
+#endif

--- a/Tests/DownViewTests.swift
+++ b/Tests/DownViewTests.swift
@@ -6,6 +6,9 @@
 //  Copyright © 2016 Glazed Donut, LLC. All rights reserved.
 //
 
+#if os(tvOS)
+    // Sorry, not available for tvOS
+#else
 import XCTest
 @testable import Down
 
@@ -103,3 +106,4 @@ fileprivate extension DownViewTests {
     }
     
 }
+#endif

